drivers/remote/api/api.go

```diff
@@ -16,6 +16,12 @@ func (r *Response) GetError() string {
 	return r.Err
 }
 
+// GetCapabilityResponse is the response of GetCapability request
+type GetCapabilityResponse struct {
+	Response
+	Scope string
```

Comment:
is it possible to add a test case for this specifically ?
I expect more changes to the capability structure and want to ensure backward compatibility for  any such changes.